src/storage/constants.rs

```diff
@@ -39,6 +38,8 @@ pub enum RootPrefix {
 
     /* Used to index user submitted username proofs */
     UserNameProofByName = 16,
+
+    OnChainEventState = 17,
```

Comment:
`NodeLocalState`

Reply:
I was thinking those can be postfixes of the same root prefix. But actually, we shouldn't do this at the `RootPrefix` enum. I think its better to do a `LocalStateRootPrefix` because it's going to be in a separate DB and these two prefixes should never collide.